src/main/python/cfn_sphere/stack_configuration/__init__.py

```diff
@@ -9,22 +9,20 @@
 class Config(object):
```

Comment:
I took a look at it: This is very impressive and utilizes a lot of "Python magic". OTOH I am not sure that everybody can read and understand the code easily. Especially looking at code like the one in the `validate_options` function makes me wonder who will understand this at the first or even second reading.

I already stumbled over the business logic in that code as it displays a JSON as example but expects a YAML as input. Just pasting the example output into the config file does not work :-(

I think that it would be best to hold of rewriting the config handling for a later time.